### PR TITLE
feat: add vue-caution/element-selectors-with-scoped

### PR DESCRIPTION
**Pull request by David-Pena**

### CODE REVIEW

**Comment by David-Pena** [comment]
I just realized I left a sample html tags const and I didn't install the package for all the tags, my bad. Tomorrow I'll add the fix to this PR so for now it's not ready to merge

**Comment by rrd108** [comment]
Also the rule checker is not included in the PR. Was it intentional?

File: docs/rules/vue-caution/element-selectors-with-scoped.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Prefer class selectors over element selectors in scoped styles, because large numbers of element selectors are slow. &nbsp;&nbsp;<br />
+👉 https://vuejs.org/style-guide/rules-use-with-caution.html#element-selectors-with-scoped
+
+### 📊 Benchmark
+
+We created a [benchmarking tool](https://github.com/David-Pena/element-class-selector-benchmark) to compare the performance of class selectors versus element selectors in JavaScript.
+
+The benchmark results clearly indicate that **class selectors consistently outperform element selectors** in terms of speed when applying styles in JavaScript. As the number of HTML elements increases, the time difference between the two selector types becomes more pronounced. Specifically:
+
+- With **10,000 HTML elements**, class selectors are approximately **17% faster** than element selectors.
```
Comment (rrd108) [inline docs/rules/vue-caution/element-selectors-with-scoped.md:12]: I would add the value in millisconds also to make it clearer

**Review by rrd108 — CHANGES_REQUESTED** [review]

**Comment by David-Pena** [comment]
> Also the rule checker is not included in the PR. Was it intentional?

I just checked and it's there in 'rulesCheck'. The rule is after the 'implicitParentChildCommunication'. Do you see it?

**Comment by rrd108** [comment]
Yes, no I see them. I think it was some temporary issue.

File: docs/rules/vue-caution/element-selectors-with-scoped.md (hunk 1 above)
Comment (David-Pena) [reply docs/rules/vue-caution/element-selectors-with-scoped.md]: added in latest commit

**Comment by David-Pena** [comment]
This PR is ready for review @rrd108

**Review by rrd108 — APPROVED** [review]

**Comment by rrd108** [comment]
released in https://github.com/rrd108/vue-mess-detector/releases/tag/v0.28.0

**Comment by rrd108** [comment]
@David-Pena  You forgot to add the rule to the sidebar.

`yarn docs:missing` helps to check this.

I added it to there and released the updtaed docs site.

It is really cool that you built a benchmarking tool for this :smile:

**Comment by David-Pena** [comment]
> @David-Pena You forgot to add the rule to the sidebar. 

😔 my bad, thanks for updating it! I'll be more cautious next time

> It is really cool that you built a benchmarking tool for this 😄

Why wait for someone else to update their benchmark 😅 thanks!

**Pull request merged**